app/src/main/java/com/app/calculator/Operator.java

```diff
@@ -2,7 +2,8 @@
 
 public enum Operator {
     DEF("def","Select Operation"),
-    ADD("add","+");
```

Comment:
Avoid hardcoded strings, you can have constants declared in the class or have an utils class that holds the constants.